hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -49,6 +51,8 @@
 
   private static final Logger LOG = LogManager.getLogger(KafkaOffsetGen.class);
 
+  private final Pattern pattern = Pattern.compile(".*,.*:.*");
```

Comment:
Please add some comments to describe the pattern? i.e. an example checkpoint string?